tracer.go

```diff
@@ -210,3 +210,16 @@ func (t *tracerImpl) Join(operationName string, format interface{}, carrier inte
 	}
 	return nil, opentracing.ErrUnsupportedFormat
 }
+
+// GetGlobalTracerOptions attempts to get the basictracer.Options
+// struct provided to the global Tracer.  Returns the options and true
+// if the global Tracer is a basictracer implementation; returns
+// false otherwise.
+func GetGlobalTracerOptions() (Options, bool) {
```

Comment:
I just took a look at the PR in the lightstep-tracer-go repo and have a much better idea what this PR is trying to accomplish. 

@jmacd How about generalizing this, so instead it's `GetTracerOptions(tracer opentracing.Tracer)...`